conf/nonclangable.conf

```diff
@@ -201,6 +201,10 @@ TOOLCHAIN:pn-gcc-for-nvcc-runtime:tegra = "gcc"
 #|       | ^
 TOOLCHAIN:pn-optee-os:imx-nxp-bsp = "gcc"
 
+# Does not compile with clang 18
+# ERROR: LLVM's thinLTO only works with gnu gold, lld, lld-link, and ld64, not ld.bfd
+TOOLCHAIN:pn-lxc = "gcc"
```

Comment:
we do not enable thin-lto by default, but its a possible DISTRO_FEATURE so it might be better to check for this in DISTRO_FEATURES and then resort to always using gcc toolchain